OmiseSDK/CardExpiryDateTextField.swift

```diff
@@ -13,26 +13,27 @@ public class CardExpiryDateTextField: OmiseTextField {
         
         return rx
     }()
-  
-    /// Currently selected month, `nil` if no month has been selected.
-    public private(set) var selectedMonth: Int? = nil
-    /// Currently selected year, `nil` if no year has been selected.
-    public private(set) var selectedYear: Int? = nil
+    
+    /// Currently selected month, `0` if no month has been selected.
+    public var selectedMonth: Int = 0
+    
+    /// Currently selected year, `0` if no year has been selected.
+    public var selectedYear: Int = 0
```

Comment:
I think it no need krub because NSNotFound search items in serial data and return indices and set default value to 0 just enough like CardExpiryDatePicker class.